### ISSUE TITLE
Error logo

### ISSUE THREAD

**Issue by charlotterazon** [marian-m12l/studio#248]
**Describe the bug**
For every story made with the STUdio, I now got an error logo when I launch them, at the very beginning, and sometimes after a few minutes (possibly at the beginning of a second node)

**To Reproduce**
1. Select a homemade story
2. Click on 'OK'
3. See error

**Desktop and environment (please complete the following information):**
 - OS: MacOS 11.6
 - Safari 15
 - Java version: the last one
 - STUdio application version [e.g. 0.3.1]

**Comment by dedesite** [comment]
I also got this error for the story I made.
I try very simple story with :
- One cover node with sound and image
- One story node with sound
The cover node properly appear but when pushing "OK" button got the error icon and no sound.

I also try with another method, with : 
- One stage node defined as start node with an image and a sound
- A simple action node (no modification on it)
- And another stage node with another sound
Same problem.

What's weird is that the sample app works and if I put the sound I want in one of the sample app stage node, it works...
I'm new to STUdio so I may be doing something wrong.
